framework/test/mocha/unit/modules/chain/submodules/rounds.js

```diff
@@ -251,282 +247,6 @@ describe('rounds', () => {
 		});
 	});
 
-	describe('__private.updateRecipientsRoundInformationWithAmountForTransactions', () => {
```

Comment:
Can we skip instead of delete so we can have a reference when implementing new tests?

Reply:
We removed this function in this PR, therefore we are removing the relevant tests for this function.